components/Tabs/src/MDCTabBar.m

```diff
@@ -280,7 +280,9 @@ - (CGSize)sizeThatFits:(CGSize)size {
   size = [_itemBar sizeThatFits:size];
 #if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
   if (@available(iOS 11.0, *)) {
-    size.height += self.safeAreaInsets.bottom;
+    if (![[self class] isTopTabsForPosition:_barPosition]) {
```

Comment:
Sorry, I missed the conversation regarding why this is needed. Why do we need to check this?